packages/cactus-plugin-ccmodel-hephaestus/src/main/typescript/pm4py-adapter/ccmodel-adapter.ts

```diff
@@ -0,0 +1,67 @@
+import { execSync } from "child_process";
+
+import {
+  CrossChainModel,
+  CrossChainModelType,
+} from "../models/crosschain-model";
+import path from "path";
+
+export function checkConformance(
```

Comment:
the idea is ok, but the input of the function should be hepheastus specific, and not files. The function can then resolve the relevant files and run the python script